lib/ansible/modules/cloud/amazon/GUIDELINES.md

```diff
@@ -496,3 +531,63 @@ other helper function `boto3_tag_list_to_ansible_dict` to get an appropriate tag
 calling this function. Since the AWS APIs are not uniform (e.g. EC2 versus Lambda) this will work
 without modification for some (Lambda) and others may need modification before using these values
 (such as EC2, with requires the tags to unset to be in the form `[{'Key': key1}, {'Key': key2}]`).
+
+## Integration Tests for AWS Modules
+
+All new AWS modules should include integration tests to ensure that any changes in AWS APIs that
+affect the module are detected. At a minimum this should cover the key API calls and check the
+documented return values are present in the module result.
+
+For general information on running the integration tests see the [Integration Tests page of the
+Module Development Guide](http://docs.ansible.com/ansible/latest/dev_guide/testing_integration.html).
+Particularly the [cloud test configuration section](http://docs.ansible.com/ansible/latest/dev_guide/testing_integration.html#other-configuration-for-cloud-tests)
+
+The integration tests for your module should be added in `test/integration/targets/MODULE_NAME`.
+
+You must have the following in `test/integration/targets/MODULE_NAME/aliases`
+
+```
+cloud/aws
+posix/ci/cloud/group4/aws
```

Comment:
group4 isn't a must - it should be one in of the (currently) 5 groups though

Reply:
Is there a rationale I should include for how to pick a group? Or is it just as simple as "we want to distribute the tests between the groups, doesn't really matter which group any given test is in" ?

Reply:
I struggle to answer that. I think I should know how to work that out (put it in the group that's currently taking the last time) but I don't know how to get that information. 

Perhaps @mattclay or @s-hertel could help answer. Your wording would certainly work in the absence of a more definitive answer.

Reply:
I recommend stating that new AWS tests should use the same group as existing AWS tests. If we can avoid a direct reference to group4 that will mean one less place to update documentation when it changes.